### PR TITLE
ISPN-14324 docs updates

### PR DESCRIPTION
**Pull request by dvagnero**
https://issues.redhat.com/browse/ISPN-14324 address multiple docs issues that were discovered during QE review

- Update pom.xml about Jakarta EE dependencies in the upgrading doc
- Update SQL store example
- Temporarily remove pipe command form the CLI quide
- specify that controlRole and monitorRole are jmx roles

needs backport to 14

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/upgrading.adoc [hunk 1]
```diff
@@ -100,14 +100,12 @@ Some modules that have Jakarta based jars are the following (note there are quit
 * infinispan-client-hotrod
 * infinispan-api
 
-If your application requires using the jakarta instead of javax namespace all you need to do is change your maven
-dependency to use the jakarta scope
+If your application requires using the jakarta instead of javax namespace all you need to do is to append your dependencies with `-javax`.
```
Comment (ryanemerson) [inline documentation/src/main/asciidoc/topics/upgrading.adoc:103]: Isn't this `-jakarta`?

Comment (dvagnero) [reply documentation/src/main/asciidoc/topics/upgrading.adoc:103]: yes, you're right. I'll fix it

Comment (dvagnero) [reply documentation/src/main/asciidoc/topics/upgrading.adoc]: Fixed

File: documentation/src/main/asciidoc/topics/upgrading.adoc [hunk 2]
```diff
@@ -100,14 +100,12 @@ Some modules that have Jakarta based jars are the following (note there are quit
 * infinispan-client-hotrod
 * infinispan-api
 
-If your application requires using the jakarta instead of javax namespace all you need to do is change your maven
-dependency to use the jakarta scope
+If your application requires using the jakarta instead of javax namespace all you need to do is to append your dependencies with `-javakarta`.
```
Comment (ryanemerson) [inline documentation/src/main/asciidoc/topics/upgrading.adoc:103]: `s/javakarta/jakarta` :slightly_smiling_face:

Comment (dvagnero) [reply documentation/src/main/asciidoc/topics/upgrading.adoc]: Oh no sorry multitasking...

**Review by karesti — APPROVED** [review]

**Comment by karesti** [comment]
@dvagnero I merge in @ryanemerson behalf

**Comment by dvagnero** [comment]
Thanks @karesti

**Pull request merged**